src/main/java/uk/gov/hmcts/reform/sendletter/entity/LetterRepository.java

```diff
@@ -2,18 +2,32 @@
 
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.jpa.repository.Lock;
+import org.springframework.data.jpa.repository.Query;
 
+import java.sql.Timestamp;
 import java.util.Optional;
 import java.util.UUID;
 import java.util.stream.Stream;
 import javax.persistence.LockModeType;
 
 public interface LetterRepository extends JpaRepository<Letter, UUID> {
+
+    String STALE_LETTERS_QUERY = "select"
```

Comment:
If the objective is to avoid loading the PDF column then instead of having an explicit query we should be able to make use of lazy loading for the pdf column using the [Lob annotation](https://docs.jboss.org/hibernate/jpa/2.1/api/javax/persistence/Lob.html) on the pdf field.

Reply:
`Lob` is not needed, apparently. It maps to different column type, `Basic` usage is enough: https://stackoverflow.com/a/3739047/984221